### DISCUSSION TITLE
Local plans (& timetables)

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#97] [Planning consideration]
# Local plans

Previously, we have done some work on what data about [development plans](https://github.com/digital-land/data-standards-backlog/discussions/26) would be useful to make available on [planning.data.gov.uk](https://www.planning.data.gov.uk/). We've also considered Local plans to be one type of development plan document, along with things like waste and mineral plans and neighbourhood plans.

However, because of the importance of Local plans we have split out the work.

This discussion has been set up so that we can update and discuss progress.

**Comment by colmjude** [comment]
## Update Oct 2024

[A video explaining why we are starting with baselining Local plan data](https://www.youtube.com/watch?v=ZMm3ykuNXf8).

**Reply by colmjude** [reply]
We have created a tool to help us review and check local plan data we have scraped from planning authority websites.

![CleanShot 2024-10-28 at 10 12 33@2x](https://github.com/user-attachments/assets/325c2543-1726-463d-b97f-79949b7eef2a)

As we check records in the tool we can mark them "for platform". This means they will be made available for the platform to collect, building up the local plan related datasets on [planning.data.gov.uk](https://www.planning.data.gov.uk). These are:

* [local plan](https://www.planning.data.gov.uk/dataset/local-plan)
* [local plan boundary](https://www.planning.data.gov.uk/dataset/local-plan-boundary)
* [local plan document](https://www.planning.data.gov.uk/dataset/local-plan-document)

**Reply by colmjude** [reply]
We are currently working on extending the tool to include timetable data. Part of this work was adding a `local plan timetable` dataset to the [local plan specification](https://digital-land.github.io/specification/specification/local-plan/).

**Comment by colmjude** [comment]
## Timetables

Timetables are the series of events that happen during the local plan-making process. The "timetable" dataset will include estimated events and actual events. This will allow users of the data to track when things are expected to happen and when things actually happened.

The [list of events can be viewed on this finder page](https://dluhc-datasets.planning-data.dev/dataset/local-plan-event/finder).

We have created a [template spreadsheet](https://docs.google.com/spreadsheets/d/15LQaAcedOn8V6VZPAhpu_G3R1qix-OPL3NesAOLxOCo/edit?usp=sharing) that you can use to create timetable data. It lists all the events and you can fill in the `event-date` and link it to a local plan. The spreadsheet contains example data for a few plans. We create this data by look at various LPAs' local development schemes or reading the page on their site that provides information on their local plan. You'll see that for some we have been able to extract a comprehensive set of data, for others it is a bit sparser. But we've collected what we can - some data is always better than no data.

We are going to use this thread to capture examples, if we find any, of timetables that LPAs publish.

**Reply by colmjude** [reply]
Thurrock have a timetable element on their [local plan site](https://consult.thurrock.gov.uk/local-plan-initial-proposals)

![thurrock-example-timetable](https://github.com/user-attachments/assets/648c21e3-44e7-4a7b-951f-f31d3e60f709)

**Reply by colmjude** [reply]
Camden had this "timetable" in their [local development scheme](https://www.camden.gov.uk/documents/20142/4855432/2023+LDS+FINAL.pdf/50d11f59-5926-6d74-d380-1bc4a30d7133?t=1702459128479)

![camden-timetable-from-lds](https://github.com/user-attachments/assets/06254ea0-333f-43f6-bdaa-052dd8f921d8)

**Comment by colmjude** [comment]
## Issue - duplicate dates

We capture the `adopted-date` on the main Local plan record.

This is useful because a user can take the local plan dataset and for any plan they will know if it has been adopted (the `adopted-date` field will have a date in it) and when it was adopted (again, the `adopted-date` field provides this information).

However, we also collect the date it was adopted as an event in the `local-plan-timetable` dataset.

This is a problem because it means we are capturing the same data in 2 places. This is something we try to avoid because if the same data is in 2 places then it opens up the possibility of them being different and therefore confusing the user of the data.

This is the case with `adopted-date` and may become a issue with other significant, status like events. Another one being when a plan is withdrawn.

**Comment by colmjude** [comment]
## Issue - is found-sound an appropriate event

When the Inspector publishes their report, they are saying that the plan is capable of being **found sound**, subject to the LPA making the agreed main modification changes.

This report marks the end of examination.

Therefore there is a question about whether we need to record both the date the inspector's report is published and the date the examination period ends.

There is also a big question mark about whether we should capture the plan being **found-sound** as a specific event because it is more an outcome of the steps in the process instead of a specific event along the way.

**Comment by colmjude** [comment]
## Updating your local plan record overtime

We had a question about how to handle changes to your local plan record over time.

The simplest way is to update the parts of the record as they change, whilst keeping things like the `reference` the same (so that the platform recognises it as being about the same plan).

For example, lets look at a plan going through the Draft → Submitted → Adopted stages.Only the `name` and, eventually, the `adopted-date` change:

1. **Draft Local Plan**  
The plan is in draft, so the LPA has titled it **“Draft Local Plan.”**


reference | name | period-start-date | period-end-date | local-plan-boundary | document-url | adopted-date | entry-date | start-date | end-date
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- 
LP-CMD-123 | Draft Local Plan | 2025-01-01 | 2035-12-31 | BND-456 | https://camden.gov.uk/plans/LP-123.pdf | | 2020-06-01 | 2020-01-01 | 

2. **Local Plan (Submitted)**  
Once submitted, they update the title so it’s clear people are looking at the submitted version.

reference | name | period-start-date | period-end-date | local-plan-boundary | document-url | adopted-date | entry-date | start-date | end-date
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- 
LP-CMD-123 | Local Plan (submitted) | 2025-01-01 | 2035-12-31 | BND-456 | https://camden.gov.uk/plans/LP-123.pdf | | 2020-06-01 | 2020-01-01 | 

3. **Adopted Local Plan**  
After adoption, they change the name again and fill in the `adopted-date` (this allows users of the data to filter on only adopted plans). Everything else stays exactly the same.

reference | name | period-start-date | period-end-date | local-plan-boundary | document-url | adopted-date | entry-date | start-date | end-date
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- 
LP-CMD-123 | Adopted Local Plan | 2025-01-01 | 2035-12-31 | BND-456 | https://camden.gov.uk/plans/LP-123.pdf | 2021-11-06 | 2020-06-01 | 2020-01-01 |

**Comment by psd** [comment]
I've added three numbers to the draft [local plan specification](https://digital-land.github.io/specification/specification/local-plan/) and [local-plan dataset](https://digital-land.github.io/specification/dataset/local-plan/) to capture the high level housing numbers in a local plan:

* `required-houses` --- the total housing requirement (net additional dwellinghouses) for the plan period
* `allocated-houses` --- the number of additional dwellinghouses allocated to sites by the local plan
* `committed-houses` -- the number of dwellinghouses already committed for development within the local plan area

where "dwellinghouse" is a building, not flats (see [designing buildings wiki](https://www.designingbuildings.co.uk/wiki/Dwellinghouse_definition) for definitions) and "committed for development" relates to permissioned developments which have commenced.

The windfall houses can therefore be calculated from `required`-`allocated`-`committed`.

**Reply by psd** [reply]
After some discussion, we're renaming these fields from `-houses` to `-dwellings` and are clarifying the precise definition of a dwelling with policy colleagues. There is discussion about certain kinds of development being classed as dwellings differently in different places. For example, some monitoring excludes student and sheltered accommodation. We are working towards the number being comparable with other definitions across the planning and housing system.

**Reply by psd** [reply]
We are also looking to explicitly record `windfall-dwellings` rather than treating it as a calculation to enable counts of other classes of allocation to be added to a plan.

**Reply by psd** [reply]
Reverting this idea: we wanted to move the housing numbers to `-dwellings` to add precision, but that doesn't capture the numbers in existing local plans and requires a policy change for future plans. So we've moved back `-housing` described as "the amount of housing" which is similar to the number of homes. We can look at possibly being more precise when it comes to recording each allocated site in the future.

**Reply by psd** [reply]
We've also created a separate [local-plan-housing](https://digital-land.github.io/specification/dataset/local-plan-housing/) dataset which enables the breakdown of housing numbers for each LPA area in a joint local plan:

* `required-housing`
* `allocated-housing`
* `committed-housing`
* `windfall-housing`
* `broad-locations-housing`

**Comment by psd** [comment]
Local plans are for a single or multiple LPA areas. We don't need to support "partial" local plans.

To simplify how we ask a LPA to provide the boundary for a local plan, I've added the `local-planning-authority` field to the `local-plan-housing` dataset.

I've also added `local-planning-authorities` field to the `local-plan` dataset. This may be used to reference the local-planning-authority area for a single plan, and the list of local planning authority areas for a joint local plan.

This is expected to replace the `local-plan-boundary` dataset.

**Comment by psd** [comment]
The draft `local-plan` dataset included an `adopted-date` field. This is an inconsistent design pattern. I've deprecated this field, replacing it with the `start-date` which is the date when the plan applies (when it is adopted) and clarified the guidance against the dataset and the specifications.

**Comment by psd** [comment]
We separated out the `local-plan-timetable` dataset to match the separation in how the policies have been described. I think this complicates understanding for users, and the importance of the reference linking the datasets together.
https://digital-land.github.io/specification/specification/local-plan/ 

<img width="767" height="750" alt="image" src="https://github.com/user-attachments/assets/ac21292d-6104-43dc-9eac-7f16965648b4" />

https://digital-land.github.io/specification/specification/local-plan-timetable/

<img width="344" height="269" alt="image" src="https://github.com/user-attachments/assets/f0ba6d18-5ad6-429f-b251-8151e83cae44" />

**Reply by psd** [reply]
The specifications now contain more content. On the timetable I've clarified the way to predict dates, adding them as fields to every timetable entry, reducing the number of different events:

<img width="1000" height="657" alt="image" src="https://github.com/user-attachments/assets/90bb27e8-3bf1-4f97-82b0-db1649bc9a87" />
